### DISCUSSION TITLE
Related Links

### DISCUSSION THREAD

**Discussion by rmccar** [ONSdigital/design-system#718] [Feedback]
Use this place to discuss the [related links component](https://ons-design-system.netlify.com/components/related-links/) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Related links top border should align with top of `page__main` heading
![Screenshot 2020-12-14 at 16 35 58](https://user-images.githubusercontent.com/43346934/102108409-cd5e7400-3e2a-11eb-902e-16c3c1915572.png)

**Comment by jrbarnes9** [comment]
This component has been deprecated and replaced by [Related content](https://ons-design-system.netlify.app/components/related-content/). This discussion will be deleted.
